index.bs

```diff
@@ -258,14 +249,17 @@ Report image Element Timing {#sec-report-image-element}
 
     1. Let |image| be |imageRequest|'s <a>associated element</a>.
     1. Let |imageResponse| be the associated <a>fetch response</a> from fetching |imageRequest|.
+    1. Let |intersectionRect| be the value returned by the <a>intersection rect algorithm</a> using |image| as the target and viewport as the root.
+    1. Let |exposed element| be the result of running <a>get an element</a> with |image| as input.
+    1. If |exposed element| is not null, call the <a>potentially add a Largest-Contentful-Paint-Candidate entry</a> algorithm with |intersectionRect|, |imageRequest|'s <a>request URL</a>, |image|'s <a attribute for=Element>element id</a>, |renderTime|, the result of running <a>get response end time</a>, and |image|.
```

Comment:
What's the context object here? We're calling 'get response end time' but that depends on the context object. It may be better to pass an object to it as a parameter instead?

Reply:
Yeah, passed in a request, as it seems sufficient